testing/unittests/jlibtests.cpp

```diff
@@ -104,6 +105,18 @@ class JlibTraceTest : public CppUnit::TestFixture
 
 protected:
 
+    void testDeclaredSpanStartTime()
+    {
+        std::chrono::nanoseconds nownanos = nowTimesStampInNanos();
+        Owned<IProperties> emptyMockHTTPHeaders = createProperties();
+
+        Owned<ISpan> serverSpan = queryTraceManager().createServerSpan("declaredSpanStartSpan", emptyMockHTTPHeaders, SpanFlags::None,  nownanos);
+
+        Owned<IProperties> retrievedSpanCtxAttributes = createProperties();
+        bool getSpanCtxSuccess = serverSpan->getSpanContext(retrievedSpanCtxAttributes, false);
+        CPPUNIT_ASSERT_EQUAL_MESSAGE("Unexpected getSpanContext failure detected", true, getSpanCtxSuccess);
```

Comment:
I'm not sure what this test is testing - it isn't using nownanos.  I think you will need to add the unit tests that look at the trace output before you can test properly.